package.json

```diff
@@ -81,14 +81,15 @@
     "eslint-plugin-react": "^7.25.1",
     "eslint-plugin-react-hooks": "^4.2.0",
     "fast-glob": "^3.2.5",
-    "husky": "^3.0.0",
+    "husky": "^4.0.0",
     "inquirer": "^6.2.2",
     "jest": "^27.0.6",
     "jest-cli": "^27.0.6",
     "jest-junit": "^12.0.0",
+    "lint-staged": "^12.1.2",
     "markdown-to-jsx": "6.11.4",
     "mkdirp": "^1.0.4",
-    "prettier": "^2.2.1",
+    "prettier": "^2.5.0",
```

Comment:
Can you try reverting prettier version changes and checking whether build passes? Maybe they did some breaking changes or added some rules that we need to override.

Reply:
Bingo, that fixed it

Reply:
Can you check what has been changed if we need to update to the newest version for some reason?

Reply:
I'm not sure. I tried to check their release notes but didn't see anything https://prettier.io/blog/